# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../smbldap-tools/fix_makefile.patch
# Copyright (C) 2004 - 2006 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---
--- ./Makefile.orig	2005-02-13 17:26:18.000000000 -0300
+++ ./Makefile	2005-02-15 11:47:03.776589016 -0300
@@ -18,10 +18,7 @@
 rm=/bin/rm
 sed=/bin/sed
 
-prep:
-	$(sed) -e 's|@SBINDIR@|$(sbindir)|g' smb.conf.in > smb.conf
-
-all:	prep distclean rpm
+all:	distclean
 
 install:
 	@mkdir -p $(sbindir)
